### PR TITLE
Upload blog link and img

### PR DESCRIPTION
**Pull request by MomtahinaKarim**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

File: _data/blogs.yml [hunk 1]
```diff
@@ -2,11 +2,11 @@
 # Please add your entries to a random position in the file.
 # This is to remove conflicts with other people adding entries.
 
-# The Format is :
-# - url: URL to the blog
-#   title: Title of the blog
-#   img: Image of the blog
-#   author_name: Name of the author of the blog (most likely you)
```
Comment (nikit19) [inline _data/blogs.yml]: You are doing it correctly just don't delete these lines of code

**Review by nikit19 — CHANGES_REQUESTED** [review]
Also upload only one picture

**Comment by nikit19** [comment]
Looks good just rename the image. The image name that you wrote in code and the image you uploaded don't have the same name

**Review by nikit19 — APPROVED** [review]
Looks good ! well done

**Comment by MomtahinaKarim** [comment]
Yes, you are right. Now, I have renamed it.

**Comment by MomtahinaKarim** [comment]
Thanks.

**Comment by MomtahinaKarim** [comment]
Thanks.

**Review by anshumanv — APPROVED** [review]
LGTM

**Pull request merged**